bucket/tageditor.json

```diff
@@ -7,11 +7,17 @@
         "64bit": {
             "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-x86_64-w64-mingw32.exe.zip",
             "hash": "ff4725dd63f5b8961dedd051c4b22f635d791f337adc2fc43c576e93c9250a5c"
+        },
+        "32bit": {
+            "url": "https://github.com/Martchus/tageditor/releases/download/v3.4.2/tageditor-3.4.2-i686-w64-mingw32.exe.zip",
+            "hash": "0163b9f1fd0cd44bb18887fbeecdd66de461113735ea704de23e54e0a4a98e6a"
         }
     },
+    "pre_install": "Get-ChildItem \"$dir\\tageditor*.exe\" | Rename-Item -NewName \"$dir\\tageditor.exe\"",
```

Comment:
- Gci is overkill. Explicitly pass the executable name to -Path
- Rename item does not take full path

https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/rename-item?view=powershell-7.1#description